.gitignore

```diff
@@ -30,3 +31,9 @@ db.sqlite3
 
 # dotenv (environment variables, used for secret/local stuff)
 .env
+
+#Name of the virtual environment
+my_project
```

Comment:
I believe pipenv automatically creates a virtual environment elsewhere (by default, in a user's .local directory) – let's switch to that, and then I think this becomes unneeded.

Reply:
Yeah it created a virtual environment like this in my case on running pipenv install requests : 
`Virtualenv location: /home/kriti/.local/share/virtualenvs/oh-proj-management-Yei1TVXn`
I think Pipfile was also created by the same command.

Reply:
Great! Sounds like you can remove this from the .gitignore, then. (If you make a new commit to this branch, your PR automatically updates.)